src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1242,6 +1242,9 @@ private function registerCacheConfiguration(array $config, ContainerBuilder $con
         $container->getDefinition('cache.adapter.system')->replaceArgument(2, $version);
         $container->getDefinition('cache.adapter.filesystem')->replaceArgument(2, $config['directory']);
 
+        if (isset($config['prefix_seed'])) {
+            $container->setParameter('cache.prefix.seed', $config['prefix_seed']);
```

Comment:
Why not setting it directly on the pass it self? Yes; foreaching all passes to find any instance of `CachePoolPass`.

Otherwise maybe use a less generic name, to clarify something's going on (it's a tmp thingy, only to transfer config..), ie. `setParameter(CachePoolPass::class.'_prefix_seed')` and avoid possible collisions in user-land.